pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -255,7 +255,14 @@ public static class Broker {
 
     public static final String CONFIG_OF_BROKER_QUERY_REWRITER_CLASS_NAMES = "pinot.broker.query.rewriter.class.names";
     public static final String CONFIG_OF_BROKER_QUERY_RESPONSE_LIMIT = "pinot.broker.query.response.limit";
-    public static final int DEFAULT_BROKER_QUERY_RESPONSE_LIMIT = Integer.MAX_VALUE;
+    public static final String CONFIG_OF_BROKER_DEFAULT_QUERY_LIMIT =
+        "pinot.broker.default.query.limit";
+
+    public static final int DEFAULT_BROKER_QUERY_LIMIT_OVERRIDE = Integer.MAX_VALUE;
```

Comment:
Let's keep the name consistent with config key, and make it next to `CONFIG_OF_BROKER_QUERY_RESPONSE_LIMIT` (i.e. keep line 258 unchanged)